mlog/global.go

```diff
@@ -25,18 +26,9 @@ func RedirectStdLog(logger *Logger) {
 
 type LogFunc func(string, ...Field)
 
-// DON'T USE THIS Modify the level on the app logger
-func GloballyDisableDebugLogForTest() {
-	globalLogger.consoleLevel.SetLevel(zapcore.ErrorLevel)
-}
-
-// DON'T USE THIS Modify the level on the app logger
-func GloballyEnableDebugLogForTest() {
-	globalLogger.consoleLevel.SetLevel(zapcore.DebugLevel)
-}
-
 var Debug LogFunc
 var Info LogFunc
 var Warn LogFunc
 var Error LogFunc
 var Critical LogFunc
+var DoWithLogLevel func(string, func())
```

Comment:
Since all of these have access to `TestHelper` receivers, instead of adding another function here, we should use `me.App.Log` and access the non-global logger instance.

Reply:
Agreed.